proto/v1/v1/database_group_service.proto

```diff
@@ -164,6 +164,9 @@ enum DatabaseGroupView {
 
   // Include everything.
   DATABASE_GROUP_VIEW_FULL = 2;
+
+  // Include matched databases.
+  DATABASE_GROUP_VIEW_MATCHED = 3;
```

Comment:
IMO, we should just return matched databases only as default (without this enum). There is no reason to show unmatched databases. Even if there is a need, users/callers can list all databases in the project minus matched databases.

Reply:
That's also what I planned. WIP